Makefile

```diff
@@ -28,17 +30,20 @@ VERSION := $(shell date '+%Y.%m.%d.%H')
 
 check: shellcheck shfmtcheck
 
-package:
+packages: $(addprefix package-,$(ALL_PLATFORMS))
```

Comment:
since we're no longer supporting "make package", and replacing it with "make packages".. we also need to update the README to reflect this.. as well as the comment towards the top of this file, and the one in `debian/rules`:
```
± g g "make package"
Makefile:# overridden by running: make package VERSION="<custom version>"
README.md:    $ ./scripts/docker-run.sh make package
debian/rules:   # infinite loop would occur: make package -> make package -> ...
```
Additionally, `.travis.yml` should be updated to reflect this.

This will also require a flag-day with the devops-gate, since we run `make package`, which will need to change to `make packages`.